### PR TITLE
Add "{status}" placeholder

### LINKED ISSUE #20: {exitcode} or {success} placeholder?
don't CIs report on either a specific `{job}` succeeded normally or failed one way or another?  I would have loved to annotate in the path to quickly see which jobs failed and which not

### PR DESCRIPTION
**Pull request by jwodder**
Closes #20.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -210,6 +211,8 @@ Placeholder     Definition
 ``{commit}``    The hash of the commit the build ran against
 ``{number}``    The run number of the workflow run (GitHub) or the build number
                 (Travis and Appveyor)
+``{status}``    The success status of the build; the exact strings used depend
```
Comment (yarikoptic) [inline README.rst:214]: could we provide `{status_harmonized}` or alike which would bring them to some common dictionary?  how different are they?

Comment (jwodder) [reply README.rst:214]: * The GitHub Actions conclusions are `action_required`, `cancelled`, `failure`, `neutral`, `success`, `skipped`, `stale`, and `timed_out`.
* I can't find documentation on the Travis states, but they include `passed`, `errored`, and `failed`.
* I can't find documentation on the Appveyor statuses, but they include `success`, `failed`, and `cancelled`.

Comment (yarikoptic) [reply README.rst:214]: > * The GitHub Actions conclusions are `action_required`, `cancelled`, `failure`, `neutral`, `success`, `skipped`, `stale`, and `timed_out`.
> * I can't find documentation on the Travis states, but they include `passed`, `errored`, and `failed`.
> * I can't find documentation on the Appveyor statuses, but they include `success`, `failed`, and `cancelled`.

Well, the main purpose is to be able to tell good from not good and the one which did not run/complete at all. 
So appveyor seems to come closest to such coarse level, but would be sad to loose the failed-vs-errored distinction.  So what about something like

```python
{ # harmonized: {original}
"success": {"success", "passed"},
"failed": {"failure", "failed"},
"errored": { "errored", "timed_out"},
"incomplete": {"action_required", "cancelled", "skipped", "stale"},
"other": { ... anything which is not above }
}
```
or may be even without `other` so if we run into some -- we have to categorize it into 4.  WDYT?

Comment (jwodder) [reply README.rst]: `{common_status}` placeholder added.

File: tinuous [hunk 2]
```diff
@@ -35,6 +35,20 @@ from yaml import safe_load
...
+    "failed": "failed",
+    "errored": "errored",
+    "timed_out": "errored",
+    "neutral": "incomplete",
+    "action_required": "incomplete",
+    "cancelled": "incomplete",
+    "skipped": "incomplete",
+    "stale": "incomplete",
+    # Error on unknown so we're forced to categorize them.
+}
```
Comment (yarikoptic) [inline tinuous:51]: if coded this way, let's add an explicit safeguard against typos in the mapped into values, smth like this:

```suggestion
}


assert set(COMMON_STATUS_MAP.values()) == {'success', 'failed', 'errored', 'incomplete'}
```

**Review by yarikoptic — CHANGES_REQUESTED** [review]
Looks great, thank you.  But let's add a safe-guard

File: tinuous (hunk 2 above)
Comment (jwodder) [reply tinuous:51]: Done.

**Comment by yarikoptic** [comment]
Just realized an ambiguity -- this the status for the entire "build" (which could have multiple jobs/workflows).
Could there also be the one per each individual job,  smth like `{job_status}`?

I am ok to merge as is, if you do not see a conflict/ambiguity to resolve while in this PR.

**Comment by jwodder** [comment]
@yarikoptic The statuses are for whatever's logs gets saved at the `path`.  For GitHub, this is the workflow run status.  For Travis and Appveyor, this is the job status.

**Comment by yarikoptic** [comment]
ah, gotcha... confirmed with looking at
```
$> ls -ld pr/*/*/*/*-*/                                                                  
drwx------ 1 yoh yoh  40 Apr 27 12:29 pr/5539/def0f2aa6673c2291b982bccb2502cea1787bc16/appveyor/5874-failed//
drwx------ 1 yoh yoh 400 Apr 27 12:29 pr/5539/def0f2aa6673c2291b982bccb2502cea1787bc16/appveyor/5874-success//
```
in my config where I assumed it to be for a build I ended up with groupping of failed-vs-success for the same build in different directories. ok, let's proceed for now -- not yet sure if I really need the "build status"

**Pull request merged**